docs/docsite/rst/reference_appendices/release_and_maintenance.rst

```diff
@@ -253,63 +260,63 @@ Dates listed indicate the start date of the maintenance cycle.
 ..      - May 2026
 ..      - | Python 3.11 - 3.13
 ..      - | Python 3.8 - 3.13
-..        | PowerShell TBD
+..        | PowerShell 5.1
 ..    * - 2.19
 ..      - May 2025
 ..      - Nov 2025
 ..      - May 2026
 ..      - Nov 2026
 ..      - | Python 3.11 - 3.13
 ..      - | Python 3.8 - 3.13
-..        | PowerShell TBD
+..        | PowerShell 5.1
 ..    * - 2.20
 ..      - Nov 2025
 ..      - May 2026
 ..      - Nov 2026
 ..      - May 2027
 ..      - | Python 3.12 - 3.14
 ..      - | Python 3.9 - 3.14
-..        | PowerShell TBD
+..        | PowerShell 5.1
 ..    * - 2.21
 ..      - May 2026
 ..      - Nov 2026
 ..      - May 2027
 ..      - Nov 2027
 ..      - | Python 3.12 - 3.14
 ..      - | Python 3.9 - 3.14
-..        | PowerShell TBD
+..        | PowerShell 5.1
 ..    * - 2.22
 ..      - Nov 2026
 ..      - May 2027
 ..      - Nov 2027
 ..      - May 2028
 ..      - | Python 3.13 - 3.15
 ..      - | Python 3.9 - 3.15
-..        | PowerShell TBD
+..        | PowerShell 5.1
 ..    * - 2.23
 ..      - May 2027
 ..      - Nov 2027
 ..      - May 2028
 ..      - Nov 2028
 ..      - | Python 3.13 - 3.15
 ..      - | Python 3.10 - 3.15
-..        | PowerShell TBD
+..        | PowerShell 5.1
 ..    * - 2.24
 ..      - Nov 2027
 ..      - May 2028
 ..      - Nov 2028
 ..      - May 2029
 ..      - | Python 3.14 - 3.16
 ..      - | Python 3.11 - 3.16
-..        | PowerShell TBD
+..        | PowerShell 5.1
 ..    * - 2.25
 ..      - May 2028
 ..      - Nov 2028
 ..      - May 2029
 ..      - Nov 2029
 ..      - | Python 3.14 - 3.16
 ..      - | Python 3.11 - 3.16
-..        | PowerShell TBD
+..        | PowerShell 5.1
```

Comment:
@jborean93 just so's we know come 2029 :-)  Do we know today that 5.1 will still be supported, or should we check with someone in core for each major ansible-core release on what  the PowerShell version should be?

Reply:
PowerShell 5.1 is the latest version that Windows ships with, even the upcoming Server 2025 release ships with 5.1 so that'll be what we need to support for the lifetime of that version. Maybe Server 20xx next will finally include PowerShell 7.x in box but until that happens we can't ever not support 5.1.